addons/ofxOsc/src/ofxOscMessage.cpp

```diff
@@ -142,6 +154,26 @@ float ofxOscMessage::getArgAsFloat( int index ) const
 }
 
 
+double ofxOscMessage::getArgAsDouble( int index ) const
+{
+    if ( getArgType(index) != OFXOSC_TYPE_DOUBLE )
+    {
+        if ( getArgType( index ) == OFXOSC_TYPE_INT32 )
+        {
+            ofLogWarning("ofxOscMessage") << "getArgAsDouble(): converting double to int32 for argument " << index;
```

Comment:
I think there's a mixup here - this function should return a double. If the arg is an int, is should cast the int to double (not the other way round). Also, why warn about it?